docs/docsite/rst/community/development_process.rst

```diff
@@ -169,8 +96,8 @@ Workflow Labels
 -  **needs_revision**: Waiting on the submitter to make changes.
 -  **shipit**: Waiting for final review by the core team for potential merge.
 
-Informational Labels
-~~~~~~~~~~~~~~~~~~~~
+Information labels
+""""""""""""""""""""
```

Comment:
Match the underline length to the line above?

Reply:
Good catch, thanks @jamescassell. Fixed.